astrocats_params.json

```diff
@@ -1,10 +1,29 @@
-{"stab_model": [["shifts", "multi-templates", {}],
-                ["msclg", "multi-templates", {"alpha":1e-5, "rho":2.5,"rho_l1":0, "do_dct_regularization":1, "dct_regularization_smooth":32, "dct_regularization_threshold":5, "sigma":0.0}]],
- "morphology_channel": 0,
- "baseline_smoothness": 300,
- "ca_channel": 1,
- "with_motion_movies": 1,
- "suff":"astrocat006",
- "motion_with_pca_denoise": 1,
- "ncpu":1
+{
+  "stab_model": [
+    [
+      "shifts",
+      "multi-templates",
+      {}
+    ],
```

Comment:
This is just an example that having all this in one line is as readable while more compact as well

Reply:
This  is what `atom-beautify` suggests:
```
{
  "stab_model": [
    ["shifts", "multi-templates", {}],
    ["msclg", "multi-templates", {
      "alpha": 1e-5,
      "rho": 2.5,
      "rho_l1": 0,
      "do_dct_regularization": 1,
      "dct_regularization_smooth": 32,
      "dct_regularization_threshold": 5,
      "sigma": 0.0
    }]
  ],
  "morphology_channel": 0,
  "baseline_smoothness": 300,
  "ca_channel": 1,
  "with_motion_movies": 1,
  "suff": "astrocat006",
  "motion_with_pca_denoise": 1,
  "ncpu": 1
}
```
I think it's a bit more sensible than `black`. Turns out, I don't like the way black formats things.